src/org/labkey/test/tests/list/ListDateAndTimeTest.java

```diff
@@ -0,0 +1,1434 @@
+import org.apache.poi.ss.usermodel.Workbook;
+import org.jetbrains.annotations.Nullable;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.query.Filter;
+import org.labkey.remoteapi.query.SelectRowsCommand;
+import org.labkey.remoteapi.query.SelectRowsResponse;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.SortDirection;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.categories.Data;
+import org.labkey.test.categories.Hosting;
+import org.labkey.test.components.CustomizeView;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.domain.DomainFormPanel;
+import org.labkey.test.pages.ImportDataPage;
+import org.labkey.test.pages.core.admin.LookAndFeelSettingsPage;
+import org.labkey.test.pages.list.EditListDefinitionPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.util.DataRegionExportHelper;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.ExcelHelper;
+import org.labkey.test.util.ListHelper;
+import org.labkey.test.util.PortalHelper;
+import org.labkey.test.util.TestDateUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+@Category({Daily.class, Data.class, Hosting.class})
+public class ListDateAndTimeTest extends BaseWebDriverTest
+{
+
+    private static final String PROJECT_NAME = "List Date And Time Test";
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("list");
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return PROJECT_NAME;
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        ListDateAndTimeTest init = (ListDateAndTimeTest)getCurrentTest();
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        log("Setup project and list module");
+        _containerHelper.createProject(PROJECT_NAME, null);
+        goToProjectHome();
+    }
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        _containerHelper.deleteProject(getProjectName(), afterTest);
+    }
+
+    @AfterClass
+    public static void resetAfterClass()
+    {
+        ListDateAndTimeTest init = (ListDateAndTimeTest) getCurrentTest();
+        init.resetSiteSettings();
+    }
+
+    public void resetSiteSettings()
+    {
+        log("Reset site settings.");
+        LookAndFeelSettingsPage.beginAt(this).reset();
+    }
```

Comment:
I don't think this will actually reset anything. The "Reset" button pops up an alert when I do it locally.
On the bright side, doing cleanup in `@After/AfterClass` is a bad idea anyway (especially if it does any navigation); furthermore, the reset button just POSTs to `admin-resetProperties.view`, which the test can easily do without any extra navigation.
The typical place to do this sort of cleanup would be to add another step to `TestScrubber.cleanSiteSettings`.

Reply:
I'll follow up with you on this feedback offline.